pkg/process/checks/process_common_test.go

```diff
@@ -316,7 +317,66 @@ func TestFormatIO(t *testing.T) {
 
 }
 
+func TestFormatNetworks(t *testing.T) {
```

Comment:
Should we also check if the `BytesRate` field is being correctly populated?

Reply:
Ah good catch, I think I worked on connection rate first then forgot to update the test 😆